src/test/regress/expected/multi_multiuser.out

```diff
@@ -378,7 +378,7 @@ INSERT INTO full_access_user_schema.t1 VALUES (1),(2),(3);
 -- creating the table should fail with a failure on the worker machine since the user is
 -- not allowed to create a table
 SELECT create_distributed_table('full_access_user_schema.t1', 'id');
-ERROR:  permission denied for database regression
```

Comment:
This change in output made me curious, and I started looking if #2575 is still valid.

Reply:
I manually tested and I see that an underprivileged user (one that can not create a schema on a database) can distribute tables now. I will try to add tests for this issue in this file